lib/active_merchant/billing/gateways/checkout_v2.rb

```diff
@@ -77,6 +77,9 @@ def store(credit_card, options = {})
           r.process do
             post = {}
             post[:currency] = options[:currency]
+            post[:payment_type] = "Recurring" if credit_card.type == "visa"
```

Comment:
hmm recurring payment only for VISA?

Reply:
I can confirm that. From the Checkout docs:

> The initial cardholder-initiated transaction of a scheduled recurring series should not be marked as "Recurring" (unless it is a Visa card, in which case it must be marked as "Recurring") and it should have merchant_initiated set to false. All subsequent payments in the series should include payment_type: "Recurring", merchant_initiated: true and the previous_payment_id.

Reply:
🆒